nixos/tests/xmpp/prosody.nix

```diff
@@ -28,19 +28,19 @@ import ../make-test.nix {
   };
 
   testScript = { nodes, ... }: ''
-    $server->waitForUnit('prosody.service');
-    $server->succeed('prosodyctl status') =~ /Prosody is running/;
+    server.wait_for_unit("prosody.service")
+    server.succeed('prosodyctl status | grep "Prosody is running"')
```

Comment:
could we do something like `'Prosody is running' in server.succeed('prosodyctl status')`

Reply:
That would not be enough to make the test fail, I think this is simpler